yajsapi/executor/index.ts

```diff
@@ -759,7 +788,11 @@ export class Executor implements ComputationHistory {
     ];
     try {
       while (services.indexOf(wait_until_done) > -1 || !done_queue.empty()) {
-        if (cancellationToken.cancelled) { work_queue.close(); done_queue.close(); break; }
+        if (cancellationToken.cancelled) {
+          work_queue.close();
+          done_queue.close();
```

Comment:
These two statements are repeated in `catch (error)` clause, it would be good to have them in one place, since both kinds of cancellation (caused by ctrl+c and by the `break` statement) should probably be handled in the same way.